### ISSUE TITLE
Getting started..

### ISSUE THREAD

**Issue by jntghua** [Milchreis/UiBooster#128]
Hi there,

I really appreciate an easy to use UI set of components and your work. Unfortunately, I could not get the components running: I added the source files to my project, then solved a missing dependency (com.bric.colorpicker that is), then saw that some 'lombok' stuff is also required.

Now, it does not compile since there is an error in 'lombok/var.java', the compiler states that 'var' is not allowed at this point since release 10.

Could you please advice how to start with this library or provide an example? Thank you very much in advance!

**Comment by Milchreis** [comment]
Hello @jntghua ,

thanks for the compliment. I am not sure if I have understood the problem correctly. Do you want to change the UiBooster code or do you want to use it in another project? Are you using Maven for the integration?

Maybe you can describe your environment and your goal a bit more. Normally you set the dependency on UiBooster as described in the [README.md](https://github.com/Milchreis/UiBooster?tab=readme-ov-file#include-to-project) and then you can use the classes.

**Comment by jntghua** [comment]
Hi Nick,

I just tried to use the UI components in my project. I'm not very experienced in Java but I got a small app working with IntelliJ (Maven) and using Processing as a library. Thank you for getting back to me, but since it's an audio app I'm building I unfortunately ran into other issues not concerning UI. Now I'm looking into C++ with JUCE lib, which seems more promising for what I need, even if there's some tough learning ahead..

Thanks again anyway and keep up the good work!
